README.md

```diff
@@ -68,22 +87,24 @@ createdb -E UTF8 -T template0 -O osmose osmose_frontend
 psql -c "CREATE EXTENSION pgcrypto" osmose_frontend
 ```
 
-As normal user, create the database tables:
+Go back to your original user and create the database tables. Replace `127.0.0.1` by your database hostname or IP if necessary
```

Comment:
It should work using unix socket, no need of IP.

Reply:
That is a workaround as postgresql may be configured to use `peer` authentication for socket-based connections. Using TCP forces using login/password authentication, and will match "production" environment setup, that is docker-based or a remote database server.

If you feel that explaining that here is better I can do that.

Reply:
Without answer I remove postgresql setup specific params/environment variables.